### PR TITLE
Multiple Item Related Mappings

### PR DESCRIPTION
**Pull request by RedstoneParadox**
Mappings related to several item classes and a rename of auj to ArmorMaterial.

p.s. Ignore the villager-related classes; I was merely deleting redundant files from my fork.

### CODE REVIEW

File: mappings/net/minecraft/item/ArmorMaterials.mapping [hunk 1]
```diff
@@ -1,11 +1,26 @@
 CLASS auk net/minecraft/item/ArmorMaterials
```
Comment (shadowfacts) [inline mappings/net/minecraft/item/ArmorMaterials.mapping:1]: Should all the armor classes be grouped together in their own package? `net/minecraft/item/armor`?

**Review by shadowfacts — CHANGES_REQUESTED** [review]
Local variables should not be mapped yet.

Also, please resolve merge conflicts.

**Comment by RedstoneParadox** [comment]
What do you mean local variables should not be mapped yet?

**Comment by shadowfacts** [comment]
Variables inside of methods should not be mapped yet as they're not properly supported across all of Fabric's toolchain.

File: mappings/net/minecraft/item/ArmorMaterials.mapping (hunk 1 above)
Comment (RedstoneParadox) [reply mappings/net/minecraft/item/ArmorMaterials.mapping:1]: In that case, so should any tool classes.

Comment (shadowfacts) [reply mappings/net/minecraft/item/ArmorMaterials.mapping:1]: I'm not certain about this. Leave them in `item` for now, this can be dealt with later.

**Comment by RedstoneParadox** [comment]
That would have been nice to know earlier.

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/item/BannerPatternItem.mapping [hunk 2]
```diff
@@ -1,5 +1,12 @@
 CLASS aup net/minecraft/item/BannerPatternItem
+	FIELD a patterns Lbnk;
```
Comment (SquidDev) [inline mappings/net/minecraft/item/BannerPatternItem.mapping]: Maybe just `pattern`, as the getter is just `getPattern()`?

**Review by SquidDev — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/BannerPatternItem.mapping (hunk 2 above)
Comment (asiekierka) [reply mappings/net/minecraft/item/BannerPatternItem.mapping]: Definitely. It's not a collection. Whoops!

**Review by asiekierka — APPROVED** [review]

**Review by SquidDev — APPROVED** [review]

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**